docs/reference/src/main/asciidoc/ri-spi.asciidoc

```diff
@@ -637,9 +631,6 @@ Servlet deployment safely.
 
 NOTE: Weld only supports JSF 1.2 and above.
```

Comment:
This just got my attention. Is that still true? I mean support for JSF 1.2. What about to update or remove this statement? Otherwise looks good.